packages/frontend/src/components/DataViz/config/CartesianChartConfiguration.tsx

```diff
@@ -28,7 +29,8 @@ export const CartesianChartConfig = ({
             <Tabs color="gray" defaultValue="data" keepMounted>
                 <Tabs.List>
                     <Tabs.Tab value="data">Data</Tabs.Tab>
-                    <Tabs.Tab value="styling">Styling</Tabs.Tab>
+                    <Tabs.Tab value="series">Series</Tabs.Tab>
+                    <Tabs.Tab value="axes">Axes</Tabs.Tab>
```

Comment:
Main ticket said to name it Display - any reason for renaming it?

Reply:
Because it has the same options now as 'Axes' in the Explorer and totally different options than 'Display' in the explorer. I was going for consistency. I can check with Katie.

Reply:
I made it 'Display' 👍